website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -229,6 +231,7 @@ Available options are:
 * `uri` (**required**): The URL of the external API to forward the request to.
 * `headers` (optional): Additional HTTP headers to be included in the request to the external API.
 * `audiencePolicy` (optional): See [configuring the audience policy](/concepts/integrate-with-your-own-api#configuring-the-audience-policy).
+* `includeUserPermissions` (optional): Whether to include an additional token claim in forwarded request containing the logged-in user permissions. See [configuring custom claims](/concepts/integrate-with-your-own-api#configuring-custom-claims)
```

Comment:
```suggestion
* `includeUserPermissions` (optional): See [configuring custom claims](/concepts/integrate-with-your-own-api#configuring-custom-claims).
```

Reply:
Updated in this commit: 088fb79b95c5c8d8d309617db4f8f9f1a47da13b